### PR TITLE
Telink B91 Entropy driver support

### PR DESCRIPTION
**Pull request by yurvyn**

### CODE REVIEW

File: boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts [hunk 1]
```diff
@@ -119,7 +124,7 @@
 		};
 		storage_partition: partition@f0000 {
 			label = "storage";
-			reg = <0x000f0000 0x00010000>;
+			reg = <0x000f0000 0x00008000>;
```
Comment (ceolin) [inline boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts:127]: is this related with this change ?

**Review by ceolin — COMMENTED** [review]
Other than the comment looks good

File: boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts (hunk 1 above)
Comment (yurvyn) [reply boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts]: No, this is not related to the current PR. This is related to an another Flash driver PR.
I was confused with possible merge conflicts, but now it is clear for me.

Removed this change from the current PR.

**Review by mbolivar-nordic — APPROVED** [review]

**Comment by yurvyn** [comment]
@ceolin,

One more approval is required.
Could you please approve the PR?

**Review by ceolin — APPROVED** [review]

**Comment by cfriedt** [comment]
@yurvyn - can you rebase and resolve conflicts?

**Comment by yurvyn** [comment]
> @yurvyn - can you rebase and resolve conflicts?

I will do it immediately once another Telink B91 IEEE802154 branch is merged.

**Review by carlescufi — APPROVED** [review]

**Comment by yurvyn** [comment]
Rebase is done. Please merge.
